### DISCUSSION TITLE
[Bug]: Frigate Plus view will reset focus to top of list of images when scrolling down page

### DISCUSSION THREAD

**Discussion by bmrowe** [blakeblackshear/frigate#13139] [Report a Bug] [answered]
### Describe the problem you are having

'm running Frigate+ on unraid. When I scroll the frigate (/plus) tab it will glitch and reset the focus to the top of the screen. Meaning, I scroll through 5-10 rows of images and it will auto scroll back up to the top of the screen. This bug is preventing me from easily scrolling down to categorize images.

In chrome debugger, I don't see any errors in the console. it does seem to happen after/during these two calls:
/api/events?limit=100&in_progress=0&is_submitted=0&has_snapshot=1
/api/events?limit=100&in_progress=0&is_submitted=0&has_snapshot=1&before=1723800275.862599

### Steps to reproduce

Go to the /plus tab, scroll through ~10+ rows of images. eventually Frigate will auto-scroll to the top row of images. 

### Version

0.14.0-da913d8

### Frigate config file

```yaml
I don't think the config matters for this
```

### Relevant log output

```shell
In chrome debugger, I don't see any errors in the console. it does seem to happen after these two calls:
/api/events?limit=100&in_progress=0&is_submitted=0&has_snapshot=1
/api/events?limit=100&in_progress=0&is_submitted=0&has_snapshot=1&before=1723800275.862599

No errors are thrown to the Frigate log
```

### Operating system

UNRAID

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

reolink cx410 x 7, 1 poe doorbell

**Comment by NickM-27** [comment]
what browser are you using? I am not able to reproduce this

**Reply by bimmerhead23** [reply]
I have seen this behavior using Safari on Mac.  I use Safari only for Frigate because h.265 needs.

**Reply by NickM-27** [reply, marked as answer]
This has been fixed, thanks 👍

**Comment by petervk** [comment]
I'm also experiencing this in Google Chrome. Do I need to wait till 0.14.1 is released for the fix?

**Reply by NickM-27** [reply]
You can use a dev build

**Reply by petervk** [reply]
I'd prefer to stay on stable. Is there an idea when the next release will be?

**Reply by NickM-27** [reply]
the 0.14.1 release has already occurred

**Reply by petervk** [reply]
Awesome! Thank you!
